pkg/minikube/minikube.go

```diff
@@ -154,6 +155,22 @@ func checkForExistingCluster() error {
 		fmt.Scanf("%s", &resp)
 		if strings.ToLower(resp) != "y" {
 			fmt.Println("Installation skipped")
+			checkKnativeNamespace := exec.Command("kubectl", "get", "namespaces")
+			output, err := checkKnativeNamespace.CombinedOutput()
+			namespaces := string(output)
+			if err != nil {
+				fmt.Println(string(output))
+				return fmt.Errorf("check existing cluster: %w", err)
+			}
+			if strings.Contains(namespaces, "knative") {
+				fmt.Print("Knative installation already exists.\nIt is advised to delete and recreate the cluster [y/N]: ")
+				fmt.Scanf("%s", &resp)
+				if strings.ToLower(resp) != "y" {
+					fmt.Println("Skipping installation")
+					installKnative = false
+					return nil
+				}
```

Comment:
if they do respond `y`, then we need to run the delete/recreate cluster bits

```suggestion
				if strings.ToLower(resp) != "y" {
					fmt.Println("Skipping installation")
					installKnative = false
					return nil
				} else {
				/* do stuff */
				}
```

Reply:
Right, will add it